Mathlib/Algebra/Group/Action/Defs.lean

```diff
@@ -156,7 +156,7 @@ class VAddAssocClass (M N α : Type*) [VAdd M N] [VAdd N α] [VAdd M α] : Prop
 /-- An instance of `IsScalarTower M N α` states that the multiplicative
 action of `M` on `α` is determined by the multiplicative actions of `M` on `N`
 and `N` on `α`. -/
-@[to_additive VAddAssocClass] -- TODO auto-translating
```

Comment:
This change looks like a standalone PR to me

Reply:
#24115